docs/source/models/cause.rst

```diff
@@ -558,6 +558,113 @@ below.
+  from the same year.
+
+  To illustrate this idea, say that there were 142,794 prevalent cases of
+  diabetes (both type I and type II) in Moldova amongst males in 2017. Here we 
+  could refer either to the point prevalence, say on January 1 2017, (note that 
+  GBD uses mid-year point prevalence when referring to prevalence) or we could 
+  consider the period prevalence, as we expect these to be approximately the 
+  same. Now, say that of those 142,794 cases, 509 remitted in 2017. This gives 
+  us the following rate:
+
+  .. math::
+
+    \frac{\text{509 cases}}{\text{142,794 person-years}} =
+    \frac{\text{0.0036 cases}}{\text{1 person-year}}
+
+  It is important to note that this is a remission rate for diabetes at all
+  ages. As diabetes follows an age pattern, this makes it seem as if diabetes
+  cases remit, on average, after :math:`\frac{1}{0.0036}\simeq 279` years--which 
+  clearly does a poor job of capturing the behavior of diabetes. This sort of description was only appropriate for diarrhea, as there is a uniform remission 
+  rate across all ages. With diabetes, remission is not guaranteed for type II,
+  and doesn't occur for type I, so remission rate does *not* tell us the average 
+  duration that an individual will experience diabetes.
+
+For both of the above rates, we think of :math:`r_1=0.2` as the probability that 
+a given case of diarrhea in the Philippies in 2017 will remitt in the next day, 
+and :math:`r_2=0.0036` as the probability that a given case of diabetes amongst 
+males in Moldova in 2017 will remitt in the next year. Notably, this is how 
+these rates are implemented in our models.
+
+  Lastly, we'll consider Tuberculosis in South Africa. Say that among 30-34 year 
+  old females, the duration of infection is 6 months on average. This gives us 
+  the following remission rate:
+
+  .. math::
+    \frac{\text{1 case}}{\text{6 person-months}}=\frac{\text{2 cases}}{\text{1 
+    person-year}}
+
+  Note that in this final example, as the duration of infection is on the same 
+  order of magnitude as the timestep, *period prevalence* is not approximated by 
+  either incidence or point prevalence. To find period prevalence, we would start 
+  with the point prevalence on January 1, 2017 (approximately 30 million cases), 
+  and add the incident cases over the course of the year (approximately 150 
+  million cases). We emphasize that this would also not tell us the prevalence 
+  of TB among this cohort on January 1, 2018, because we also need to account 
+  for mortality in both the susceptible and infected groups over the course of 
+  the year, in addition to remission. This is thus a less helpful measure in 
+  this example.
+
+So, to which population does the remission rate apply? For the purposes of our 
+model, the *remission rate* will be the probability that a given prevalent case 
+who does *not* die during a given timestep remitts.
```

Comment:
@collijk i made this up---i'm don't actually know how you handle this or deal with mortality vs. remission. Can we either talk next coworking or office hours, or could you correct here?

Reply:
EDIT: this was too convoluted, simplified/removed this section